run/core/awscli/test.sh

```diff
@@ -1306,6 +1398,7 @@ main() {
     test_serverside_encryption && \
     test_serverside_encryption_get_range && \
     test_serverside_encryption_multipart && \
+    test_serverside_encryption_multipart-copy && \
```

Comment:
```suggestion
    test_serverside_encryption_multipart_copy && \
```

Isn't the dash invalid here?